### PR TITLE
[minor]: SCA

### PR DESCRIPTION
**Pull request by kalessil**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | n/a

### CODE REVIEW

File: framework/db/ActiveRelationTrait.php [hunk 1]
```diff
@@ -353,24 +353,22 @@ private function populateInverseRelation(&$primaryModels, $models, $primaryName,
...
-        } else {
-            if ($this->multiple) {
-                foreach ($primaryModels as $i => $primaryModel) {
-                    foreach ($primaryModel[$primaryName] as $j => $m) {
-                        if ($m instanceof ActiveRecordInterface) {
-                            $m->populateRelation($name, $primaryModel);
-                        } else {
-                            $primaryModels[$i][$primaryName][$j][$name] = $primaryModel;
-                        }
+        } else if ($this->multiple) {
```
Comment (rob006) [inline framework/db/ActiveRelationTrait.php]: `else if` -> `elseif`?

Comment (kalessil) [reply framework/db/ActiveRelationTrait.php]: Updated

File: framework/db/oci/Schema.php [hunk 2]
```diff
@@ -621,7 +621,7 @@ public function insert($table, $columns)
                 } else {
                     $returnParams[$phName]['dataType'] = \PDO::PARAM_INT;
                 }
-                $returnParams[$phName]['size'] = isset($columnSchemas[$name]) && isset($columnSchemas[$name]->size) ? $columnSchemas[$name]->size : -1;
+                $returnParams[$phName]['size'] = isset($columnSchemas[$name], $columnSchemas[$name]->size) ? $columnSchemas[$name]->size : -1;
```
Comment (rob006) [inline framework/db/oci/Schema.php]: Can we simplify `isset($columnSchemas[$name], $columnSchemas[$name]->size)` to `isset($columnSchemas[$name]->size)`? First check seems to be redundant.

Comment (kalessil) [reply framework/db/oci/Schema.php]: I noted the case for evaluation (and an inspection enhancement), but ATM can not say.

Comment (samdark) [reply framework/db/oci/Schema.php]: Yes, we can. Could be made into inspection.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**